lib/quiet_opener.rb

```diff
@@ -30,23 +27,17 @@ def quietly_try_to_open(url, timeout=60)
   return result
 end
 
-# On Ruby versions before 2.0, we need to use the net-http-local gem
+# On Ruby versions before 2.0, we needed to use the net-http-local gem
 # to force the use of 127.0.0.1 as the local interface for the
 # connection.  However, at the time of writing this gem doesn't work
 # on Ruby 2.0 and it's not necessary with that Ruby version - one can
 # supply a :local_host option to Net::HTTP:start.  So, this helper
 # function is to abstract away that difference, and can be used as you
 # would Net::HTTP.start(host) when passed a block.
 def http_from_localhost(host)
```

Comment:
I can't see this method even being called – could we remove it entirely?

Reply:
I couldn't find where it was being called from either ... I'll have a quick look online while waiting for specs to run (if neither of those turn up anything interesting, I'll remove it)

Reply:
(Got a clean spec run, no clues online so rebasing to remove that method before merge)